seaborn/tests/test_axisgrid.py

```diff
@@ -432,7 +459,7 @@ def test_set_axis_labels(self):
         npt.assert_array_equal(got_x, xlab)
         npt.assert_array_equal(got_y, ylab)
 
-    def test_subplot_kws(self):
+    def test_axis_lims(self):
```

Comment:
The name predates `subplot_kws` as an actual argument, but yes your change is more clear. :+1: